products/mssharepoint.md

```diff
@@ -56,11 +60,15 @@ releases:
     eol: 2017-10-10
     latest: "12.0.6690.5000"
     latestReleaseDate: 2014-05-13
+    link: https://learn.microsoft.com/lifecycle/products/microsoft-office-sharepoint-server-2007
```

Comment:
```suggestion
    link: http://support.microsoft.com/help/2837616
```

Reply:
Are Windows SharePoint Services 3.0 and SharePoint Server 2007 the same product ?

It does not seem from what I can see on https://learn.microsoft.com/en-us/lifecycle/products/microsoft-windows-sharepoint-services-30.